index.html

```diff
@@ -1,52 +1,3336 @@
+            class="q7"
+          ><div>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.</div>
+<div>If you like what you see and have a project you need coded, don’t hesitate and contact me.</div>
+          </div>
+          </div>
+          <div class="outer_q8">
+          <img
+            id="q8"
+            data-name="Group 19"
+            src="images/Group_19.png"
+            alt="Group 19"
+            class="q8"
+          >
+          </div>
+        </div>
+        </div>
+        <div class="outer_q9">
+        <img
+          id="q9"
+          data-name="image_geometry_2"
+          src="images/image_geometry_2.png"
+          alt="image_geometry_2"
+          class="q9"
+        >
+        </div>
+        <div class="outer_q10">
+        <div
+          id="q10"
+          data-name="Frame 1"
+          class="q10"
+        >
+        </div>
+        </div>
+        <div class="outer_q11">
+        <div
+          id="q11"
+          data-name="Group 119"
+          class="q11"
+        >
+          <div class="outer_q12">
+          <div
+            id="q12"
+            data-name="Projects"
+            class="q12"
+          ><div>Projects</div>
+          </div>
+          </div>
+          <div class="outer_q13">
+          <div
+            id="q13"
+            data-name="Group 70"
+            class="q13"
+          >
+            <div class="outer_q14">
+            <div
+              id="q14"
+              data-name="Group 91"
+              class="q14"
+            >
+              <div class="outer_q15">
+              <div
+                id="q15"
+                data-name="Rectangle 21"
+                class="q15"
+              >
+              </div>
+              </div>
+              <div class="outer_q16">
+              <div
+                id="q16"
+                data-name="Rectangle 22"
+                class="q16"
+              >
+              </div>
+              </div>
+              <div class="outer_q17">
+              <div
+                id="q17"
+                data-name="Mobile Portfolio"
+                class="q17"
+              ><div>Mobile Portfolio</div>
+              </div>
+              </div>
+              <div class="outer_q18">
+              <div
+                id="q18"
+                data-name="Group 24"
+                class="q18"
+              >
+                <div class="outer_q19">
+                <div
+                  id="q19"
+                  data-name="Rectangle 27"
+                  class="q19"
+                >
+                </div>
+                </div>
+                <div class="outer_q20">
+                <div
+                  id="q20"
+                  data-name="JavaScript"
+                  class="q20"
+                ><div>JavaScript</div>
+                </div>
+                </div>
```

Comment:
Kindly implement the change requested earlier by wrapping the languages, frameworks and skills in`li` elements. It is recommended to use these tags when referring to items in a list group.